src/main/resources/templates/cypher/list.html

```diff
@@ -32,8 +32,7 @@ <h2>Šifry</h2>
         <td align="center">
             <a href="#" th:href="@{'/hint/list?cypherId='+${cypher.cypherId}}">Detail</a> |
             <a href="#" th:href="@{'/cypher/update/'+${cypher.cypherId}}">Upravit</a> |
-            <a href="#" th:href="@{'/cypher/delete/'+${cypher.cypherId}}">Smazat</a> |
-            <a href="#" th:href="@{/progress/(cypherId=${cypher.cypherId})}">Stav</a>
```

Comment:
Šlo by ještě na téhle stránce přepsat ty linky na tenhle formát? @{/progress/(cypherId=${cypher.cypherId})}

Všiml jsem si, že je to i na ostatních stránkách, ale to bych přepsal postupně. Nejspíš jsem si při minulých review nevšiml.

Reply:
upravil jsem to i v ostatnich strankach

Reply:
teda krome stage.html - tam jsou ty dva paramery v requestu - to jsem nestihl :)